sbi/samplers/mcmc/__init__.py

```diff
@@ -4,7 +4,7 @@
     resample_given_potential_fn,
     sir_init,
 )
-from sbi.samplers.mcmc.slice import Slice
+from sbi.samplers.mcmc.pymc_utils import PyMCSampler
```

Comment:
since there is no other pymc file, I would change the name to `pymc.py` or `pymc_wrapper.py`